spec/lib/email/receiver_spec.rb

```diff
@@ -415,9 +415,43 @@ def create_post_reply_key(value)
     it "automatically elides gmail quotes" do
       SiteSetting.always_show_trimmed_content = true
       expect { process(:gmail_html_reply) }.to change { topic.posts.count }
-      expect(topic.posts.last.raw).to eq(
-        "This is a **GMAIL** reply ;)\n\n<details class='elided'>\n<summary title='Show trimmed content'>&#183;&#183;&#183;</summary>\n\nThis is the *elided* part!\n\n</details>",
-      )
+      expect(topic.posts.last.raw).to eq <<~MD.strip
```

Comment:
This is not part of the fix, but I used a heredoc string to make it easier to read.